src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -165,3 +166,24 @@ To use CodeStream's Open in IDE integration with your APM stack trace errors, us
 Once set up, you can jump from an error group directly to the offending code in your IDE by clicking the **Open in IDE** button.
 
 Learn more [here](docs/codestream/start-here/codestream-new-relic/#apm).
+
+## Connect an inbox to Jira [#jira]
+
+Connect errors inbox to Jira to easily create tickets for your errors, allowing for faster collaboration and resolution. Templates allow you to quickly create a ticket containing error details and links directly to the stack trace and APM for quick access and resolution. 
+
+We store a link to the the ticket alongside the associated error group for a period of time. If the error occurs again within that period, you can easily access closed associated tickets for context.
+
+To connect an inbox to Jira:
+1. Click on the Jira integration icon on the far right side of the error group you want to connect to Jira. 
+  ![An image showing the location the jira integration button in errors inbox.](./images/jiraint.png "Jira integration button")
+  <figcaption>Clicking the Jira integration icon allows you to create a ticket based on a template, or create a templat if you don't already have one.</figcaption>
+2. If you don’t already have a [connection to Jira](/docs/alerts-applied-intelligence/notifications/notification-integrations/#set-up-jira) set up in your account, click **Add JIRA Workspace** from the dropdown. Fill in all the field and **Click Test** connection before saving to ensure that your details are correct.
+3. Next, set up a template. Templates determine what information will be sent to Jira. Find more information about specific fields [here](/docs/alerts-applied-intelligence/notifications/notification-integrations/#configure-jira-message-template). Errors inbox does not currently support two-way communication with Jira, but you can select this option and have it automatically enabled when we roll it out.
```

Comment:
"when we roll it out" is a bit of a promise, which we generally avoid. Maybe just say "if we support it." Or "in case it is supported."